project/MimaExcludes.scala

```diff
@@ -60,6 +60,10 @@ object MimaExcludes {
               "org.apache.spark.ml.regression.LeastSquaresCostFun.this"),
             ProblemFilters.exclude[MissingMethodProblem](
               "org.apache.spark.ml.classification.LogisticCostFun.this"),
+            ProblemFilters.exclude[MissingMethodProblem](
```

Comment:
That will work. But it's private class, and it's only used here, so I don't think it's necessary to preserve the original signature.